community/kernel/src/main/java/org/neo4j/kernel/NeoStoresDiagnostics.java

```diff
@@ -47,7 +47,7 @@ void dump( NeoStores source, Logger logger )
         @Override
         void dump( NeoStores source, Logger log )
         {
-            source.getMetaDataStore().logRecords( log );
+            source.getMetaDataStore().logRecords( log::log );
```

Comment:
as i have already ask, as well as Konstantin please change it to be Log instead of Consumer